profiling-ffi/src/crashtracker/datatypes.rs

```diff
@@ -89,7 +89,7 @@ impl<'a> TryFrom<CrashtrackerConfiguration<'a>>
             vec
         };
         let create_alt_stack = value.create_alt_stack;
-        let endpoint = unsafe { Some(exporter::try_to_endpoint(value.endpoint)?) };
+        let endpoint = unsafe { exporter::try_to_endpoint(value.endpoint).ok() };
```

Comment:
What does it mean for the crashtracker to have an empty endpoint ? We can add some comments to ensure the ffi API is clear.

Reply:
Added 

>   /// If ProfilingEndpoint is left to a zero value (enum value for Agent + empty charslice),
    /// the crashtracker will infer the agent host from env variables.